frontend/schema/class-schema.php

```diff
@@ -27,6 +27,15 @@ public function register_hooks() {
 	 * @since 1.8
 	 */
 	public function json_ld() {
+		/**
+		 * Filter: 'wpseo_json_ld_output' - Allows disabling Yoast's schema output entirely.
+		 *
+		 * @api mixed If false or an empty array is returned, disable our output.
+		 */
+		$return = apply_filters( 'wpseo_json_ld_output', 'test' );
```

Comment:
Would the implementations not expect an array to be passed as the argument?

Reply:
So far only the "woo-seo-addon"-plugin extends on it, but it does expect an array to be passed (hard type implementation on the underlaying call).

Suggesting for value to pass:

`array( '_deprecated' => 'please use "wpseo_schema_*" filters to extend the Yoast SEO schema data - see the WPSEO_Schema class' )`

Reply:
Sounds good. Can you take care of that?